priv/test/migrations/20160929125415_migrations.exs

```diff
@@ -1,22 +0,0 @@
-defmodule Guardian.DB.Test.Repo.Migrations do
```

Comment:
Why do you remove this file?

Reply:
This was per needing to wanting to auto-create tables and migrations. But upon refactoring to use different repo, I will re-add this. However, removing it did result in failures as pointed out in my notes.